### ISSUE TITLE
Removal of Cooliris Integration

### ISSUE THREAD

**Issue by Chraneco** [JoomGallery/JoomGallery#14]
Cooliris development has been discontinued by its maintainers and the public API is already not available anymore.

Thus, everything related to Cooliris will be removed from JoomGallery's code base.

It doesn't seem like many people used this feature lately anyway.
If you rely on some code of this or suggest a workaround please post here before the release of JoomGallery 3.3. Otherwise, Cooliris will be removed with that release.

**Comment by JoshuaLewis** [comment]
I'm in favor of removing Cooliris. Yahoo acquired Cooliris, but isn't doing anything with it.

**Comment by Erftralle** [comment]
:+1: 
I created pull request #28 to remove Cooliris.

**Comment by Erftralle** [comment]
Since we have a merged pull request for this issue we could close here, right?

**Comment by Chraneco** [comment]
GitHub should automatically close this issue once the 3.3 branch is merged into master. I'd like to test that feature.
